recipes/sfepy/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set rev = "d479f7a9" %}
+{% set name = "sfepy" %}
+{% set org = "sfepy" %}
+{% set version = "d479f7a9" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/{{ org }}/{{ name }}.git
```

Comment:
IMHO yes (not tested) - it should point to the same commit as the ``release_{{ version }}`` tag.